contracts/colonyNetwork/ColonyNetworkMining.sol

```diff
@@ -240,8 +241,8 @@ contract ColonyNetworkMining is ColonyNetworkStorage {
   }
 
   function burnUnneededRewards(uint256 _amount) public stoppable onlyReputationMiningCycle() {
-    ITokenLocking(tokenLocking).claim(IMetaColony(metaColony).getToken(), true);
-    ITokenLocking(tokenLocking).burn(_amount);
+    address clnyToken = IMetaColony(metaColony).getToken();
+    ERC20Extended(clnyToken).burn(_amount);
```

Comment:
I am overly suspicious of this change, but haven't investigated fully. I can't immediately see why the tokens are now in a different place.

Reply:
It is due to the above change removing `makeConditonalDeposit` -- now all reclaimed stakes are held by the mining cycle directly.